flight_planning/v1/astm_f3548v21.yaml

```diff
@@ -1,14 +1,18 @@
 openapi: 3.0.2
 info:
-  title: Definitions used in flight planning interface specific to strategic conflict detection, but not using data structures aligned with ones defined in ASTM F3548-21.
+  title: Definitions used in flight planning interface specific to ASTM F3548-21.
+  version: 0.0.1
```

Comment:
I don't completely follow.  I think I agree that referencing external models for an API can cause some problems since a change in the dependency can be a breaking change to the API (if that is a problem you are suggesting may occur?).

The main reason for including a version number is that it is required by OpenAPI.  Without that version, some tooling may break.  Plus we won't have a clear way to track the version of the model without it.

A couple related notes: 

1. I think OpenAPI 3.1 allows for these kinds of "model-only" files.  3.0 requires paths or something to be true to spec, so we may still be breaking OpenAPI specs. 
2. One can also reference a raw JSON schema file from an OpenAPI document if we only need models.  Just an idea... it relieves some of the OpenAPI requirements for better or worse.  Not sure it's a good idea, but it's a potential option.

I don't have super-strong feelings on all of the above, but I do like meeting standards.

Reply:
I don't care very much about this, but I do want to make sure we remain compatible with as much tooling as possible and also convey accurate information.  While the OpenAPI 3.0 spec doesn't appear to require particular behavior from versions (such as semver rules), I'm concerned that something might break or be confusing if we intentionally have two different file contents with the same version (e.g, `local`).  I also agree we don't want to maintain component versions -- only the top API is something we intend to maintain versioning on, and the component files are just making it easier to manage.  So, I think that leaves us with making the component files not OpenAPI files.  I'll remove the OpenAPI headers so they are no longer OpenAPI documents.  Please let me know if anyone thinks this might break any tooling (I'm pretty sure it won't break InterUSS tooling).

Reply:
The referenced files are now no longer OpenAPI documents.